ConnectorRhino/ConnectorRhino7/ConnectorRhino7.csproj

```diff
@@ -44,7 +44,7 @@
     <ProjectReference Include="..\..\DesktopUI2\DesktopUI2\DesktopUI2.csproj" />
   </ItemGroup>
   <ItemGroup Condition="'$([MSBuild]::IsOSPlatform(OSX))' == 'true'">
-    <None Include="$(HOME)/.nuget/packages/avalonia.native/0.10.17/runtimes/osx/native/libAvaloniaNative.dylib" CopyToOutputDirectory="PreserveNewest" />
+    <None Include="$(HOME)/.nuget/packages/avalonia.native/0.10.18/runtimes/osx/native/libAvaloniaNative.dylib" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
would a wildcard work here? So we do not forget updating this in the future...

Reply:
Not sure it would... I think we should actually be using this:

https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#generatepathproperty